integration/flink/src/test/java/io/openlineage/flink/FlinkContainerUtils.java

```diff
@@ -0,0 +1,188 @@
+
+import com.google.common.io.Resources;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.List;
+import lombok.SneakyThrows;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.MockServerContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.lifecycle.Startable;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+public class FlinkContainerUtils {
+
+  static MockServerContainer makeMockServerContainer(Network network) {
+    return new MockServerContainer(
+            DockerImageName.parse("jamesdbloom/mockserver:mockserver-5.12.0"))
+        .withLogConsumer(of -> consumeOutput("mockserver", of))
+        .withNetwork(network)
+        .withNetworkAliases("openlineageclient");
+  }
+
+  static GenericContainer<?> makeSchemaRegistryContainer(Network network, Startable startable) {
+    return genericContainer(network, "confluentinc/cp-schema-registry:6.2.1", "schema-registry")
+        .withExposedPorts(28081)
+        .withEnv("SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS", "PLAINTEXT://kafka:9092")
+        .withEnv("SCHEMA_REGISTRY_HOST_NAME", "schema-registry")
+        .withEnv("SCHEMA_REGISTRY_LISTENERS", "http://schema-registry:8081,http://0.0.0.0:28081")
+        .dependsOn(startable);
+  }
+
+  static GenericContainer<?> makeKafkaContainer(Network network, Startable zookeeper) {
+    return genericContainer(network, "confluentinc/cp-kafka:6.2.1", "kafka")
+        .withExposedPorts(9092, 19092)
+        .withEnv(
+            "KAFKA_LISTENER_SECURITY_PROTOCOL_MAP",
+            "LISTENER_DOCKER_INTERNAL:PLAINTEXT,LISTENER_DOCKER_EXTERNAL:PLAINTEXT")
+        .withEnv(
+            "KAFKA_ADVERTISED_LISTENERS",
+            "LISTENER_DOCKER_INTERNAL://kafka:9092,LISTENER_DOCKER_EXTERNAL://127.0.0.1:19092")
+        .withEnv("KAFKA_INTER_BROKER_LISTENER_NAME", "LISTENER_DOCKER_INTERNAL")
+        .withEnv("KAFKA_ZOOKEEPER_CONNECT", "zookeeper:2181")
+        .withEnv("KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "1")
+        .withEnv("KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS", "0")
+        .withEnv("KAFKA_TRANSACTION_STATE_LOG_MIN_ISR", "1")
+        .withEnv("KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", "1")
+        .withEnv("KAFKA_AUTO_CREATE_TOPICS_ENABLE", "true")
+        .withEnv("KAFKA_BROKER_ID", "1")
+        .withEnv(
+            "KAFKA_LOG4J_LOGGERS",
+            "kafka.controller=INFO,kafka.producer.async.DefaultEventHandler=INFO,state.change.logger=INFO")
+        .dependsOn(zookeeper);
+  }
+
+  @SneakyThrows
+  static GenericContainer<?> makeInitKafkaContainer(Network network, Startable kafka) {
+    return genericContainer(network, "confluentinc/cp-kafka:6.2.1", "init-kafka")
```

Comment:
Also, would be nice to have image names in static variables, so we can change the versions in one place.